tensorflow-core/tensorflow-core-generator/src/main/java/org/tensorflow/processor/operator/OperatorProcessor.java

```diff
@@ -154,17 +155,39 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
   }
 
   private static class OpsSpec {
+    private static final Comparator<MethodSpec> PARAMETER_SPEC_COMPARATOR = (o1, o2) -> {
+      if (o1.parameters.size() > o2.parameters.size()) {
+        return 1;
+      } else if (o1.parameters.size() < o2.parameters.size()) {
```

Comment:
no need of `else` blocks if you return in the previous `if`

Reply:
Ok, I'll remove it.